nixnet/_enums.py

```diff
@@ -1674,12 +1674,22 @@ class Phase(enum.Enum):
 
 
 class DevForm(enum.Enum):
-    '''Device physical form factor.'''
+    '''Device physical form factor.
+
+    Values:
+        PXI
+        PCI
+        C_SERIES
+        PXIE
+        USB
+        PCIE
+    '''
     PXI = _cconsts.NX_DEV_FORM_PXI
     PCI = _cconsts.NX_DEV_FORM_PCI
     C_SERIES = _cconsts.NX_DEV_FORM_C_SERIES
     PXIE = _cconsts.NX_DEV_FORM_PXIE
     USB = _cconsts.NX_DEV_FORM_USB
+    PCIE = _cconsts.NX_DEV_FORM_PCIE
```

Comment:
Very minor. it might help if we sorted the values by bus type or alphabetically (both in code and in docstring) from legibility standpoint.

For example:

C_SERIES
PCI
PCIE
PXI
PXIE
USB

Super minor so not going to hold the review for this.

Reply:
I agree; sorted is better and there is no need to keep these values in the same order as the C API. Done.